imageio/imageio-tiff/src/main/java/com/twelvemonkeys/imageio/plugins/tiff/CCITTFaxDecoderStream.java

```diff
@@ -106,6 +111,15 @@ public CCITTFaxDecoderStream(final InputStream stream, final int columns, final
                 "CCITT GROUP 3/4 OPTION UNCOMPRESSED is not supported");
     }
 
+    /**
+     * This is used for CCITT streams from PDF files, which use EncodedByteAlign
+     *
+     * @param b EncodedByteAlign
+     */
+    public void setOptionByteAligned(boolean b) {
```

Comment:
It cannot change while decoding. For normal CCITT decoding as in TIFF compressions it should never be changed from the default: false for G3/G4, true for MODIFIED HUFFMAN RLE. Changing this flag from default is only a special case for PDF CCITT, which is why I would not like it as an additional constructor parameter.